invoices/invoiceregistry.go

```diff
@@ -659,8 +659,9 @@ func (i *InvoiceRegistry) cancelSingleHtlc(invoiceRef InvoiceRef,
 
 		// Only allow individual htlc cancellation on open invoices.
 		if invoice.State != ContractOpen {
-			log.Debugf("cancelSingleHtlc: invoice %v no longer "+
-				"open", invoiceRef)
+			log.Debugf("CancelSingleHtlc: cannot cancel htlc %v "+
+				"on invoice %v, invoice is no longer open", key,
```

Comment:
nit: think we wanna be as concise as possible in the logging - so i would do,
`"CancelSingleHtlc: cannot cancel htlc: invoice(%v) no longer open"`